it/content/src/main/content/jcr_root/content/forms/sites/core-components-it/aemembedformstest/.content.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+          xmlns:fd="http://www.adobe.com/aemfd/fd/1.0"
+          jcr:primaryType="cq:Page">
+    <jcr:content
+            cq:lastModified="{Date}2023-04-18T12:30:56.377+05:30"
+            cq:lastModifiedBy="admin"
+            cq:tags="[]"
+            cq:template="/conf/core-components-examples/settings/wcm/templates/content-page"
+            jcr:description="Aem Embed Container With Custom Height"
+            jcr:primaryType="cq:PageContent"
+            jcr:title="Aem Embed Container With Custom Height"
```

Comment:
Can you rename the collateral to `embedFormInIframe` and `embedFormInNonIframe` ?

Reply:
Removed AEM inside the collateral for changing the name of this will require changes at multiple places will do that as a incrementally.  Also the new test cases are added with the exact name.